aws/logs_monitoring/lambda_function.py

```diff
@@ -19,6 +19,19 @@
 
 import boto3
 
+try:
```

Comment:
Existing users may not have lambda layer added to their existing log forwarder installations, the new installations of log forwarder are expected to have the layer added.